lib/internal/process.js

```diff
@@ -283,6 +283,53 @@ function setupUncaughtExceptionCapture(exceptionHandlerState) {
   };
 }
 
+function setupCompareVersion() {
+  const {
+    majorVersion,
+    minorVersion,
+    patchVersion,
+  } = process.release;
+  const prereleaseTag = process.release.prereleaseTag.replace(/^-/, '');
+
+  process.release.compareVersion =
+    (major, minor, patch, prerelease = false) => {
```

Comment:
What should happen if all of or some of `major, minor, patch` are undefined? Should the function throw or should the arguments be `0`? Currently, it seems they are `NaN` in this case.